packages/account/src/test-utils/launchNode.ts

```diff
@@ -185,6 +185,7 @@ export const launchNode = async ({
         ['--ip', ipToUse],
         ['--port', portToUse],
         useInMemoryDb ? ['--db-type', 'in-memory'] : ['--db-path', tempDir],
+        ['--tx-ttl-check-interval', '1s'],
```

Comment:
Why add this to the `launchNode` code? Shouldn't it be only added to `transaction-response.test.ts` tests as they were the ones failing?

Reply:
Probably wise.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/3272/commits/73094a1b91a3b28996024d57e470ea1dfdf9edf5